Makefile

```diff
@@ -1,56 +1,24 @@
-BIN := kube-mgmt
-PKG := github.com/open-policy-agent/kube-mgmt
-REGISTRY ?= openpolicyagent
-VERSION := 0.12.1
-ARCH := amd64
-COMMIT := $(shell ./build/get-build-commit.sh)
-
-IMAGE := $(REGISTRY)/$(BIN)
-
-BUILD_IMAGE ?= golang:1.12-alpine
-
-.PHONY: all
-all: image
+IMAGE := openpolicyagent/kube-mgmt
+VERSION ?= $(shell git describe --tags --dirty | sed 's/^v//g')
```

Comment:
Nit: Move this in a script in the `build` folder with a brief description of what it does.

Reply:
Didnt understant the purpose, pleae repharse.

Reply:
We keep scripts in the `build` folder. We can add a new file in there with this command and an explanation of what it does. Not a blocker though.

Reply:
Because, those script are not used anywherre outside release pupeline. There is no need to pollute the project with unnecessary files. Instead al release logic is moved to makefile and pipeline.